CONTRIBUTING.md

```diff
@@ -312,6 +312,41 @@ Alternatively, the `make test_ci` target can be used instead of
 `make test` in order to run in a setup that replicates the configuration
 we used in our CI systems more closely.
 
+### Snapshot Testing for Visualizations
+
+If you are working on code that makes changes to any matplotlib visualisations 
```

Comment:
our plan is to add circuit latex drawer to this schema at some point. So I think "matplotlib" is a bit too narrow. But "visualization" is a bit too wide (for example, circuit text drawer is a visualization that wont be not included).
 
```suggestion
If you are working on code that makes changes to rich visualisations 
```

Is that too obscure?

Reply:
The rest of the explanation is very much focus on circuit matplotlib visualization. So probably is better as it is and then come back here to update as you tests are being added.

```suggestion
If you are working on code that makes changes to the circuit matplotlib drawer 
```